### ISSUE TITLE
Update suggestions for known issues

### ISSUE THREAD

**Issue by Hitchcock3** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#3989]
In the section where moving from SfB to Teams is discussed (https://docs.microsoft.com/en-us/microsoftteams/teams-for-vdi#migrate-from-skype-for-business-on-vdi-to-teams-on-vdi) there is a mention that "Screen share from chat without audio" is not currently supported. However this is not included in the list of Known Issues and Limitations (https://docs.microsoft.com/en-us/microsoftteams/teams-for-vdi#known-issues-and-limitations) This is confusing as most would consider this a known issue or limitation but it is not included in this section. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 17880b9d-6b98-3194-cc33-fe8067ce241e
* Version Independent ID: 643bcf2f-507b-e449-73db-923d8a368ad9
* Content: [Teams for Virtualized Desktop Infrastructure - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/teams-for-vdi#migrate-from-skype-for-business-on-vdi-to-teams-on-vdi)
* Content Source: [Teams/teams-for-vdi.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/teams-for-vdi.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by scanum** [comment]
@Hitchcock3  Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by lindspea** [comment]
@Hitchcock3 thank you for reaching out to help improve MS Docs. I've just looked through the doc and seen that it has been updated 2 days ago and it looks like the limitation has been included under knows issues and limitations - 
"Outgoing screen sharing: 
Application sharing is not supported.
Give control and take control: 
Not supported during a screen sharing or application sharing session."
Let me know if I can help further.
